Common/src/main/java/com/hypherionmc/sdlink/util/PKUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.hypherionmc.sdlink.util;
+
+import com.hypherionmc.sdlink.core.discord.BotController;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.jetbrains.annotations.Nullable;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Objects;
+
+public class PKUtil {
+    public static final String PK_APP_ID = "466378653216014359";
+    public static final HashSet<String> PK_USERS = new HashSet<>();
```

Comment:
```suggestion
    public static final Set<String> PK_USERS = Collections.synchronizedSet(new HashSet<>());
```
This HashSet needs to be thread safe.